index.bs

```diff
@@ -3423,6 +3423,33 @@ examples that violate the above rules are {{XMLHttpRequest}} and
 initialisms, even if they are repeated.
 </div>
 
+<h4 id="factory-naming" class="no-num no-toc">Start factory method names with `create` or `from`</h4>
+
+When defining a new factory method, prefix its name with either `create` or `from`.
+
+If a factory method constructs a new empty object,
+prefix the method name with `create`.
+However, if your factory method creates an object from existing data,
+prefix the method name with `from`.
+
+Factory methods <a href="#constructors">are an exception</a>, and not the norm.
+An example of valid usage of a factory method is
+when an object is being created it also requires association
+with the parent object.
+`document.createXXX()` is an example of this.
```

Comment:
I think this is actually an antipattern that we should actively discourage. This should have been `Element.create({ document })`, because:
1. It’s a pattern that does not scale: what happens when you need to associate with two objects?
2. It does not make it clear what object is created. With factory methods being static methods on a class, even when a subclass is returned, at least we know what the base class is, whereas something like `document.createXXX()` could return anything.
3. It introduces an undesirable coupling between two classes when all we want is a relationship. E.g. what if in the future we want elements that are not associated with a document? 
4. It’s violates most precedent. 

Furthermore, the way this is phrased is inaccurate: for DOM elements the parent object is `element.parentNode`, not `document`. Which highlights another problem with this pattern.